test-utils/chain-spec-builder/Cargo.toml

```diff
@@ -5,7 +5,10 @@ authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2018"
 
 [dependencies]
-clap = { version = "~2.32.0", features = ["yaml"] }
+ansi_term = "0.12.1"
+clap = { version = "~2.32.0" }
+keystore = { package = "substrate-keystore", path = "../../core/keystore" }
 node-cli = { path = "../../node/cli" }
 primitives = { package = "substrate-primitives",  path = "../../core/primitives" }
-substrate-service = { path = "../../core/service" }
+rand = "0.7.2"
+structopt = "0.2.0"
```

Comment:
```suggestion
structopt = "0.3.3"
```

Use latest.

Reply:
The reason I did this was because we're using `structopt = "0.2.0"` elsewhere, and I didn't want to add a new version. Might make sense to update structopt everywhere if it's easy.